<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>320</width>
      <height>180</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>&amp;Roman To Decimal</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>192</width>
        <height>99</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="romanNumberLineEdit">
         <property name="text">
          <string>Type roman numeral...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="convertRomToDecButton">
         <property name="text">
          <string>Convert Roman To Decimal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="romToDecResultLabel">
         <property name="text">
          <string>Result</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>&amp;Decimal To Roman</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>192</width>
        <height>99</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLineEdit" name="decimalNumberLineEdit">
         <property name="text">
          <string>Type decimal number...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="convertDecToRomButton">
         <property name="text">
          <string>Convert Decimal To Roman</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="decToRomResultLabel">
         <property name="text">
          <string>Result</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
